2023-08-18 11:06:23,840 INFO    MainThread:47553 [internal.py:wandb_internal():90] W&B internal server running at pid: 47553, started at: 2023-08-18 11:06:23.839981
2023-08-18 11:06:23,842 DEBUG   HandlerThread:47553 [handler.py:handle_request():141] handle_request: run_start
2023-08-18 11:06:23,842 INFO    WriterThread:47553 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230818_110623-13gznpyl/run-13gznpyl.wandb
2023-08-18 11:06:23,843 DEBUG   HandlerThread:47553 [meta.py:__init__():35] meta init
2023-08-18 11:06:23,843 DEBUG   HandlerThread:47553 [meta.py:__init__():49] meta init done
2023-08-18 11:06:23,844 DEBUG   HandlerThread:47553 [meta.py:probe():209] probe
2023-08-18 11:06:23,844 DEBUG   HandlerThread:47553 [meta.py:_setup_git():199] setup git
2023-08-18 11:06:23,851 DEBUG   HandlerThread:47553 [meta.py:_setup_git():206] setup git done
2023-08-18 11:06:23,851 DEBUG   HandlerThread:47553 [meta.py:_save_pip():53] save pip
2023-08-18 11:06:23,852 DEBUG   HandlerThread:47553 [meta.py:_save_pip():67] save pip done
2023-08-18 11:06:23,852 DEBUG   HandlerThread:47553 [meta.py:probe():247] probe done
2023-08-18 11:06:32,461 WARNING MainThread:47553 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-18 11:06:32,614 WARNING MainThread:47553 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-18 11:06:32,614 ERROR   MainThread:47553 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-18 11:06:32,845 INFO    SenderThread:47553 [sender.py:finish():1111] shutting down sender
2023-08-18 11:06:33,462 INFO    WriterThread:47553 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230818_110623-13gznpyl/run-13gznpyl.wandb
2023-08-18 11:06:33,463 INFO    HandlerThread:47553 [handler.py:finish():806] shutting down handler
2023-08-18 11:06:33,464 INFO    MainThread:47553 [internal.py:handle_exit():80] Internal process exited
